### ISSUE TITLE
Training loss for stage 1？

### ISSUE THREAD

**Issue by Qmebius** [Boese0601/MagicDance#57]
Thank you for your work. I would like to ask, in the first stage, is the training loss calculated as the loss between the generated image and the image in the target pose, or is it the loss with respect to Iref
